scripts/databases/seed.sh

```diff
@@ -3,23 +3,44 @@
 set -euo pipefail
 
 GROUP=$1
+shift
+DATABASE_VERSIONS=$*
 
 function createDatabase() {
-  echo "Downloading database... $1"
+  echo "Downloading database $1 ..."
+
+  mkdir -p "$HOME/Downloads"
   curl -C - "$2" -o "$HOME/Downloads/$1.sql.gz"
-  #curl "$2" -o "$HOME/Downloads/$1.sql.gz"
 
-  echo "Login..."
+  echo "Login ..."
+  rm .access_token_cache # to make sure we're not using an expired token if we're seeding a lot of databases
   source ./auth.sh
 
-  echo "Uploading database...$1"
+  echo "Uploading database $1 ..."
   ./upload.sh "$GROUP" "$HOME/Downloads/$1.sql.gz"
+  echo # empty line to improve output readability
 }
 
-## https://databases.dhis2.org/
-createDatabase "Sierra Leone - 2.39.0" https://databases.dhis2.org/sierra-leone/2.39.0/dhis2-db-sierra-leone.sql.gz
-createDatabase "Sierra Leone - 2.38.1" https://databases.dhis2.org/sierra-leone/2.38.1/dhis2-db-sierra-leone.sql.gz
-createDatabase "Sierra Leone - 2.37.7" https://databases.dhis2.org/sierra-leone/2.37.6/dhis2-db-sierra-leone.sql.gz
-createDatabase "Sierra Leone - 2.36.11" https://databases.dhis2.org/sierra-leone/2.36.0/dhis2-db-sierra-leone.sql.gz
+if [[ -z "$*" ]]; then
+  DATABASE_VERSIONS=("dev")
+
+  # Filter out all supported versions - current + 2 previous, including patch versions (like 2.40, 2.40.0, 2.39, 2.39.0, etc)
+  dhis2_versions=$(curl -fsSL https://releases.dhis2.org/v1/versions/stable.json)
+  latest_version=$(echo "$dhis2_versions" | jq -r '.versions[] | select(.latest == true) | .version')
+  # shellcheck disable=SC2207
+  DATABASE_VERSIONS+=($(
+    echo "$dhis2_versions" |
+    jq -r "[.versions[] |
+    select(.version == $latest_version or .version == ($latest_version - 1) or .version == ($latest_version - 2)) |
+    [.name] + [(.patchVersions[] | select(.hotfix != true) | .name)]] | flatten | .[]"
+  ))
+fi
+
+echo "Seeding the following databases:"
+echo "${DATABASE_VERSIONS[@]}"
```

Comment:
```suggestion
echo "Seeding the following databases: ${DATABASE_VERSIONS[@]}"
```

Reply:
I split it into two lines because of https://github.com/koalaman/shellcheck/wiki/SC2145, but I'll just disable the check for that line. 👍